### ISSUE TITLE
cannot serialize a bytes object larger than 4 GB? how to do 

### ISSUE THREAD

**Issue by wll18** [lshiwjx/2s-AGCN#35]

**Comment by kdavila** [comment]
Same issue here. Any luck?

**Comment by bqFirst** [comment]
You can try to change batch_size

**Comment by kdavila** [comment]
> You can try to change batch_size

Yes. I had to reduce it to 8 for the NTU RGB+D dataset, and it works. Thanks

**Comment by JoshaPan** [comment]
when I reduce the batch_size to 8 in train_joint.yaml
the result is as follow:
ForkingPickler(file, protocol).dump(obj)
OverflowError: cannot serialize a bytes object larger than 4 GiB

dose anyone can solve it?
